.travis.yml

```diff
@@ -3,6 +3,7 @@ cache: pip
 python:
 - '3.5'
 - '3.6'
+- '3.7-dev'
```

Comment:
👍 I can start using 3.7, if needed.

Reply:
it's release candidate will be published on the 30th.  We need a 3.7 wheel, for starters.